src/Network/GRPC/LowLevel/Server.hs

```diff
@@ -154,14 +154,12 @@ addPort server conf@ServerConfig{..} =
            C.serverAddSecureHttp2Port server e creds
   where e = unEndpoint $ serverEndpoint conf
 
-startServer :: GRPC -> ServerConfig -> IO Server
+startServer :: GRPC -> ServerConfig -> IO (Server, Port)
```

Comment:
What is the reason for not adding a field to `data Server` called `serverPort` for the assigned port information?